### DISCUSSION TITLE
Meme version features

### DISCUSSION THREAD

**Discussion by strubium** [CleanroomMC/Cleanroom#218] [Draft]
### Summary

Various "gimmick" features from newer versions  

### Goals

Add the "oh that's pretty cool, moving on" features from modern versions of Minecraft

Examples:
- https://minecraft.wiki/w/Commands/tick
  - /tick commands are really cool and fun to play with
- /spreadplayers <maxHeight> argument https://minecraft.wiki/w/Commands/spreadplayers
  - Adding in 1.16,  specify the maximum height players should be spread
- /warp
  - April Fools Day attrition, warp the player to a dimension based on the ID. https://minecraft.wiki/w/Commands/warp
- Game tests 
- Brain System? (This should have its own proposal honestly)
- Seamless terrain blending

### Non Goals

- Backport items
- Change vanilla behavior (Ex: add support for changing it, be that in configs or commands) 

### Motivation

A discussion in the Cleanroom Discord

### Description

Backport or remake the nice-to-have features from newer versions, the ones people take about when saying how "modern mc is so much better! look at all this stuff they added! its all QoL!" 

### Dependencies

Kinda a sub-proposal of adding the tag system, although not really related 

### Guidelines

- [X] I have and will continue to adhere to the guidelines

**Comment by Rongmario** [comment]
Something to keep in mind with the Brain System, as usual Mojang overcomplicated it like DFU, here's a better [model](https://github.com/Tslat/SmartBrainLib/wiki).
